content/rs/administering/import-export/importing-data.md

```diff
@@ -125,15 +125,15 @@ Before you specify to import from OpenStack Swift, make sure that you have:
 
 ### Azure Blob Storage
 
-Before you choose to backup to OpenStack Swift, make sure that you have:
```

Comment:
I believe "backup to" should also be changed to "import from" because this is the imrrot page, not the backup page. (See the OpenStack section above)

Reply:
Fixed.